dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import {
+  InputGroup,
+  InputGroupText,
+  DatePicker,
+  isValidDate,
+  yyyyMMddFormat,
+  Button,
+} from "@patternfly/react-core";
+import moment from "moment";
+
+function DatePickerWidget({
+  dataArray,
+  setPublicData,
+  controllerName,
+  setDateRange,
+}) {
+  const [fromDate, setFromDate] = useState(
+    moment(new Date(1990, 10, 4)).format("YYYY/MM/DD")
+  );
+  const [toDate, setToDate] = useState(
+    moment(new Date(2040, 10, 4)).format("YYYY/MM/DD")
+  );
```

Comment:
I suppose the idea is that by default we're showing all datasets. In theory that could be a *lot* of data. It might make more sense by default to show only datasets from the most recent month, or even a few weeks? Since you're defining a Redux store, I'm assuming that reloading the page will remember your previously selected range. (We could go further and persist that into a user preference, once the server has that capability ... but that might be going too far.)

Reply:
> I suppose the idea is that by default we're showing all datasets. In theory that could be a _lot_ of data.

True...but Shubham's going to put in pagination to handle that....  🙃

> It might make more sense by default to show only datasets from the most recent month, or even a few weeks?

Yep.

> Since you're defining a Redux store, I'm assuming that reloading the page will remember your previously selected range. (We could go further and persist that into a user preference, once the server has that capability ... but that might be going too far.)

That might make sense for the "from-date"; however, I expect that a user would want the "to-date" to include his/her most recent upload...so, using "today" seems like a good default.  (Remembering what s/he wanted last time might be handy, but it also might be the opposite.)